.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8', '11', '13', '14' ]
```

```suggestion
        java: [ '8', '11', '13' ]
```

Just FYI. The higher support version by the gradle 6.0.1 is JDK 13. Support for JDK 14 starts from the gradle 6.3+, however and the same time, starting from gradle 6.1+ there is no support for JDK 1.6 which is a baseline for this project

---

LGTM